modin/core/io/text/csv_glob_dispatcher.py

```diff
@@ -54,6 +54,11 @@ def _read(cls, filepath_or_buffer, **kwargs):
         # Ensures that the file is a string file path. Otherwise, default to pandas.
         filepath_or_buffer = cls.get_path_or_buffer(filepath_or_buffer)
         if isinstance(filepath_or_buffer, str):
+            if "*" not in filepath_or_buffer:
```

Comment:
Glob can accept a single file. Do we not want to support this case here?

Reply:
We want to support. This is just a reminder to use wildcard if needed.